Help Desk process

```diff
@@ -0,0 +1,31 @@
+Process for Help Desk Requests from External Users
+
+When a new issue is received from an external user, it automatically creates a JIRA ticket.
+Help Desk managers send a generic response (within one day) to the user notifying them that we will get them an answer soon
```

Comment:
I liked Dan's suggestion, so I created a JIRA ticket for this (automatic replies to user input): https://atlassian.kbase.us/browse/KBASE-2194